### PR TITLE
WIP hqq-mix perf tests

### PR DESCRIPTION
**Pull request by HDCharles**
Summary:

running into an error, unsure if its caused by the unwrap_tensor_subclass or what

with unwrap: I can't call nn.functional.linear because its missing detach, but inserting all the detaches in the world doesn't fix it

without unwrap: I can't torch.compile it
Test Plan:
python test/test_integration.py -k
"test_int4_weight_only_hqq_mixed_quant_subclass_api_grouped"

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: torchao/dtypes/affine_quantized_tensor.py [hunk 1]
```diff
@@ -509,9 +509,9 @@ def __torch_dispatch__(cls, func, types, args, kwargs):
     def get_plain(self):
         from torchao.quantization.quant_primitives import (
             ZeroPointDomain,
-            unpack_tinygemm_scales_and_zeros,
             quantize_affine,
         )
+        from torchao.quantization.utils import unpack_tinygemm_scales_and_zeros
```
Comment (jerryzh168) [inline torchao/dtypes/affine_quantized_tensor.py:514]: fixed in https://github.com/pytorch/ao/pull/471
